### PR TITLE
Add doc of extrinsic parameter calibration of Astra

### PR DESCRIPTION
**Pull request by pazeshun**

### CODE REVIEW

File: doc/tips_faq.rst [hunk 1]
```diff
@@ -88,3 +88,43 @@ This gripper has two parts made by 3D printers. "base" part is made by ProJet an
...
+.. code-block:: bash
+
+  % roslaunch jsk_2016_01_baxter_apc astra_hand.launch left_first:=false
+
+If the point cloud and the model of Baxter arm are too different in Rviz, you should change the position of depth optical frame like below:
+
+.. code-block:: bash
+
+  % /opt/ros/indigo/lib/tf/static_transform_publisher -0.10 -0.008 0.015 -1.56 0.00 -0.08 right_hand right_hand_camera_depth_optical_frame 100 __name:=right_hand_camera_depth_static_tf_publisher  # This is only example
+  # You can get this form like below
```
Comment (knorth55) [inline doc/tips_faq.rst]: `form` -> `node_name` or `__name` ?

File: doc/tips_faq.rst [hunk 2]
```diff
@@ -88,3 +88,43 @@ This gripper has two parts made by 3D printers. "base" part is made by ProJet an
...
+
+.. image:: _media/check_astra_raw_point_cloud.jpg
+
+If you want to reverse right and left camera vision:
+
+.. code-block:: bash
+
+  % roslaunch jsk_2016_01_baxter_apc astra_hand.launch left_first:=false
+
+If the point cloud and the model of Baxter arm are too different in Rviz, you should change the position of depth optical frame like below:
```
Comment (knorth55) [inline doc/tips_faq.rst]: `position` -> `pose`
`model of Baxter` -> `robot model` or `Baxter model`

File: doc/tips_faq.rst [hunk 3]
```diff
@@ -88,3 +88,43 @@ This gripper has two parts made by 3D printers. "base" part is made by ProJet an
...
+
+.. code-block:: bash
+
+  % /opt/ros/indigo/lib/tf/static_transform_publisher -0.10 -0.008 0.015 -1.56 0.00 -0.08 right_hand right_hand_camera_depth_optical_frame 100 __name:=right_hand_camera_depth_static_tf_publisher  # This is only example
```
Comment (knorth55) [inline doc/tips_faq.rst]: `this is only example` -> `this is just an example`

Comment (knorth55) [inline doc/tips_faq.rst]: why not `rosrun tf static_transform_publisher ~~~~` ?

Comment (pazeshun) [inline doc/tips_faq.rst]: You can get `/opt/ros/indigo/lib/tf/static_transform_publisher -0.10 -0.008 0.015 -1.56 0.00 -0.08 right_hand right_hand_camera_depth_optical_frame 100 __name:=right_hand_camera_depth_static_tf_publisher` by `roslaunch jsk_2016_01_baxter_apc astra_hand.launch --args /right_hand_camera_depth_static_tf_publisher`.
I think copying and pasting is easy way.

**Comment by pazeshun** [comment]
Fixed

**Comment by knorth55** [comment]
:+1:

**Pull request merged**